pkgpanda/__init__.py

```diff
@@ -572,9 +595,16 @@ def __init__(
                 self.__systemd_dir = "{}/dcos.target.wants".format(root)
             else:
                 self.__systemd_dir = "/etc/systemd/system/dcos.target.wants"
+
         self.__manage_systemd = manage_systemd
         self.__block_systemd = block_systemd
 
+        if rooted_systemd:
+            self.__dcos_service_configuration_json = self._make_abs("dcos-service-configuration.json")
+        else:
+            self.__dcos_service_configuration_json = os.path.join("/opt/mesosphere/",
```

Comment:
That's because  I wanted to factor out the name of the file `"dcos-service-configuration.json"`.  I will do that.